PayButton/paybutton-server
# Feat/370 improve price sync since beginning

Test Plan:
1. To test building all prices from scratch, run (warning, this will reset your entire local database):  yarn docker mr -f
      + you should see a new file created: prisma/seeds/prices.csv, it has all prices since the beginning until today (this file will be created only once)
      + maybe you see some lines “Problem getting price of XEC 20210606 -> Error: timeout of 30000ms exceeded (attempt 2)”, this is only a problem if you see attempt 5, any lesser will be retried.
      + you can query your local database and see all thousands of prices there: select * from paybutton.Price
2. To test building prices from your file you can run again:  yarn docker mr -f
      + It will run much faster and you won’t see any attempt to reach coin dance api
3. To test the price worker sync, you can open the prices.csv, delete the 3 last XEC lines (each line is a day) and:
    1. Run again: yarn docker mr -f, this will recreate your database, 3 last days will be missing, you can verify that querying the database:
        select id, FROM_UNIXTIME(p.`timestamp`), p.value, p.quoteId, p.networkId 
        from paybutton.Price p 
        where networkId = 1
        order by p.`timestamp` desc
    2. Run the worker, 1st terminal: yarn docker jr (run the worker), 2nd terminal: yarn docker jw (see the worker output)
    3. Run query in 1 again, you will see the prices there

With both the big first pull and the worker that runs every some seconds, we will always have all the price data ready in our database
